java/ql/automodel/publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e
+
+AUTOMODEL_ROOT="$(dirname $0)"
+WORKSPACE_ROOT="$AUTOMODEL_ROOT/../../../.."
```

Comment:
```suggestion
WORKSPACE_ROOT="$AUTOMODEL_ROOT/../../.."
```

Reply:
I have always been using `readlink -f` do you know if `realpath` is ok on all the places this could be thought to run?